src/Symfony/Component/Cache/Adapter/RedisAdapter.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter;
+
+/**
+ * @author Aurimas Niekis <aurimas@niekis.lt>
+ */
+class RedisAdapter extends AbstractAdapter
+{
+    /**
+     * @var \Redis
+     */
+    private $redis;
+
+    /**
+     * @param \Redis $redisConnection
+     * @param string $namespace
+     * @param int    $defaultLifetime
+     */
+    public function __construct($redisConnection, $namespace = '', $defaultLifetime = 0)
```

Comment:
Sorry, but I do not want to include all 3rd party libraries just to show possible types, but documentation is a better thing I think. Or I will be creating a library which will provide unified interface and wrapper which would wrap those objects and provide interface